proto/credentials.proto

```diff
@@ -46,14 +48,14 @@ message KeySecret {
 }
 
 message AWSSessionTokenSecret {
-  string key = 1;
-  string secret = 2;
-  string sessionToken = 3;
+  string key = 1 [(validate.rules).string.min_len = 1];
+  string secret = 2 [(validate.rules).string.min_len = 1]; 
+  string sessionToken = 3 [(validate.rules).string.min_len = 1];
```

Comment:
nit: I know this is already like this, but could we convert this to `snake_case` to conform to the [protobuf style guide](https://protobuf.dev/programming-guides/style/#message-field-names). This shouldn't need any changes to the codebase so it's a safe change.